src/app/collective-rewards/rewards/backers/hooks/useGetBackerRewards.ts

```diff
@@ -39,31 +39,28 @@ export const useGetBackerRewards = (
   currency = 'USD',
 ) => {
   const { builders, isLoading: buildersLoading, error: buildersError } = useBuilderContext()
-  const activeBuilders = filterBuildersByState<CompleteBuilder>(builders)
-  const gauges = activeBuilders.map(({ gauge }) => gauge)
+  const { activeBuilders, gauges } = useMemo(() => {
+    const filteredBuilders = filterBuildersByState<CompleteBuilder>(builders)
+    const builderGauges = filteredBuilders.map(({ gauge }) => gauge)
+    return { activeBuilders: filteredBuilders, gauges: builderGauges }
+  }, [builders])
   const {
```

Comment:
Should I move this hook to the table as it is done in the builders table?

Reply:
no, the hook is left in the builders table only because it will need to be replaced by a proper fetching mechanism that allows filtering upon query.